components/InputScanner.js

```diff
@@ -36,21 +36,54 @@ class InputScanner extends React.Component {
           <Text>Camera permission is not granted</Text> :
           <BarCodeScanner
               onBarCodeRead={this._handleBarCodeRead}
-              style={{ height: '100%', width: 500 }}
-          />
+              style={{ height: '90%', width: 400 }}
+              >
+              <View style={styles.layerTop} />
+                <View style={styles.layerCenter}>
+                  <View style={styles.layerLeft} />
+                  <View style={styles.focused} />
+                  <View style={styles.layerRight} />
+                </View>
```

Comment:
I thought we were creating a CSS folder?

Reply:
I think it was discussed in one of our code reviews that for React Native it is convention for these to be kept within the same file.